### PR TITLE
Upgrade pnotify

### PR DESCRIPTION
**Pull request by collin**
work in progress

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -94,13 +96,15 @@
...
+    <script src="/pnotify/dist/pnotify.animate.js"></script>
+    <script src="/pnotify/dist/pnotify.buttons.js"></script>
+    <script src="/pnotify/dist/pnotify.callbacks.js"></script>
+    <script src="/pnotify/dist/pnotify.confirm.js"></script>
+    <script src="/pnotify/dist/pnotify.js"></script>
+    <script src="/pnotify/dist/pnotify.desktop.js"></script>
+    <script src="/pnotify/dist/pnotify.history.js"></script>
+    <script src="/pnotify/dist/pnotify.mobile.js"></script>
+    <script src="/pnotify/dist/pnotify.nonblock.js"></script>
     <script src="/raven-js/dist/raven.js"></script>
```
Comment (collin) [inline app/index.html]: Upgrading and downgrading PNotify seemed to get us some sort of asset change here, but it still works.

File: app/scripts/app.js [hunk 2]
```diff
@@ -73,10 +73,12 @@ angular.module('bulbsCmsApp', [
...
     '$provide', '$httpProvider', '$locationProvider', '$routeProvider', '$sceProvider',
       'TokenAuthConfigProvider', 'TokenAuthServiceProvider', 'CmsConfigProvider',
-      'COMPONENTS_URL', 'PARTIALS_URL', 'FirebaseConfigProvider',
+      'COMPONENTS_URL', 'PARTIALS_URL', 'FirebaseConfigProvider', 'PNotify',
     function ($provide, $httpProvider, $locationProvider, $routeProvider, $sceProvider,
         TokenAuthConfigProvider, TokenAuthServiceProvider, CmsConfigProvider,
-        COMPONENTS_URL, PARTIALS_URL, FirebaseConfigProvider) {
+        COMPONENTS_URL, PARTIALS_URL, FirebaseConfigProvider, PNotify) {
+
+			PNotify.prototype.options.styling = 'fontawesome';
```
Comment (collin) [inline app/scripts/app.js]: This is the jam, right here.

**Comment by collin** [comment]
@kand @spra85 @mparent61 @shawncook review plz

**Comment by kand** [comment]
:+1:

**Comment by mparent61** [comment]
:+1:

**Pull request merged**